### PR TITLE
Fixed incorrect migration

### PR DESCRIPTION
**Pull request by CommanderStorm**
## Issue

Resolves: #1480 

## Screenshot
image

## Why this is useful for all students
Idk, less bugs and stuff

### CODE REVIEW

**Review by Bentipa — APPROVED** [review]
The bug is fixed, although I am not sure if there would also be a way to get that view binding to work without putting the included layout directly into the main layout file.

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/search/SearchFragment.kt [hunk 1]
```diff
@@ -4,7 +4,9 @@ import android.app.SearchManager
 import android.content.Context
 import android.content.Intent
 import android.os.Bundle
+import android.view.LayoutInflater
 import android.view.View
+import android.view.ViewGroup
```
Comment (CommanderStorm) [inline app/src/main/java/de/tum/in/tumcampusapp/component/ui/search/SearchFragment.kt:9]:
```suggestion
import android.view.View
```

Comment (CommanderStorm) [inline app/src/main/java/de/tum/in/tumcampusapp/component/ui/search/SearchFragment.kt:9]:
```suggestion
import android.view.View
```

**Comment by Bentipa** [comment]
@CommanderStorm yeah looks good :D

**Pull request merged**